app/services/jobs/return-logs/return-version-migration.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+/**
+ * Determines what licences need to have new return versions created
```

Comment:
This should match the first line of the JSDoc for the `go()` function. Not fussed which is picked!